Classes/Service/MauticSendFormService.php

```diff
@@ -71,6 +88,16 @@ private function makeHeaders(): array
             $headers['Client-Ip'] = $ip;
         }
 
+        $extensionConfiguration = GeneralUtility::makeInstance(YamlConfiguration::class)->getConfigurationArray();
+        $authorizeService = GeneralUtility::makeInstance(MauticAuthorizeService::class);
+        if ($extensionConfiguration['authorizeMode'] !== YamlConfiguration::OAUTH1_AUTHORIZATION_MODE) {
```

Comment:
Maybe you should consider to introduce a dedicated method for this since this condition is used in multiple classes. e.g. `$foo->isOAuth1Authorization()`